Crypto/Crypto/Presentation/Views/HomeView/HomeView.swift

```diff
@@ -31,18 +34,27 @@ struct HomeView: View {
                         "ARSChipColor"} getBackgroundColor: { SelectionButtonData in
                             "ChipSelectedBackgroundColor"
                         } onChipTapped: {
-                            Void.self
+                            homeViewModel.toggleDarkMode()
                         }
                     
                     UnselectedChipButton<SelectionButtonData>(item: SelectionButtonData(label: "Dark Theme", letterColor: "ARSChipColor",backgroundColor: "ChipSelectedBackgroundColor")){SelectedChipButton in "Dark Theme"} getLetterColor: { SelectionButtonData in
                         "ChipUnselectedBorder"} getBackgroundColor: { SelectionButtonData in
                             "ChipSelectedBackgroundColor"
                         } onChipTapped: {
-                            Void.self
+                            homeViewModel.toggleDarkMode()
```

Comment:
Are you also supporting dark mode with the device configuration?

Reply:
It changes the mode in all the app, but I will check if iit takes de mobile configuration

Reply:
It changes with the mobile configuration, I checked in my phone